src/test/java/model/RacingTest.java

```diff
@@ -0,0 +1,58 @@
+package model;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class RacingTest {
+
+    @DisplayName("경주에 참가할 자동차 세팅 테스트")
+    @ParameterizedTest
+    @ValueSource(strings = {"als,qweop,skld", "car1,car2,car3"})
+    void newRacingTest(String input) {
+        Racing racing = new Racing(input);
+        String racingCars = racing.getCars().stream()
+            .map(Car::getName)
+            .collect(Collectors.joining(","));
+
+        List<Car> cars = Arrays.stream(input.split(","))
+            .map(Car::new)
+            .collect(Collectors.toList());
+        String settedCars = cars.stream()
+            .map(Car::getName)
+            .collect(Collectors.joining(","));
+
+        assertThat(racingCars).isEqualTo(settedCars);
+    }
+
+    @DisplayName("자동차 게임 진행 테스트")
+    @ParameterizedTest
+    @CsvSource(value = {"car1,car2:5", "carr1,carr2:3"}, delimiter = ':')
+    void raceTest(String input, String inputMoveTimes) {
+        Racing racing = new Racing(input);
+        int moveTimes = Integer.valueOf(inputMoveTimes);
+        for (int i = 0; i < moveTimes; i++) {
+            racing.race();
+        }
+        assertThat(
+            racing.getCars().stream()
+                .map(Car::getPosition)
+                .allMatch(position -> 0 <= position && position <= moveTimes))
+            .isTrue();
```

Comment:
```suggestion
        assertThat(racing.getCars())
                .filteredOn(car -> car.getPosition() < 0 || car.getPosition() > 9)
                .isEmpty();
```

``assertJ`` api를 적극적으로 활용해보면 어떨까요? 제가 사용한 방식보다도 분명 더 좋은 방법이 있을것이라 생각이 듭니다. AssertJ api에 대해 자세히 찾아본다면? 더 간결하게 테스트를 작성하실 수 있을 겁니다.

Reply:
`AssertJ api` 에 대해서 말씀해주셨습니다. 이 부분은 저도 동의하는 부분으로 api에 대한 학습이 부족했던 것 같습니다. 보완하도록 노력하겠습니다.